api_views/test_func.py

```diff
@@ -1,8 +1,12 @@
 import requests
 
 
-def mock_bad_implementation(r: str, p: int, t: str):
-    a = f"https://api.github.com/repos/{r}/pulls/{p}"
-    h = {"Authorization": "Bearer " + t}
+def mock_bad_implementation(r: str, p: int, g: str):
```

Comment:
Function parameters 'r', 'p', 'g' should have more descriptive names to improve code readability